tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example_raw_scalars/static/model.js

```diff
@@ -14,42 +14,48 @@
 // ==============================================================================
 
 // Generic example of a service that provides data.
-// In a production plugin, consider caching these results, to avoid making
-// unnecessary requests for data..
+
+/** @type {?Object<string, !Array<string>>} */
+let runToTagInfo = null;
+
+async function updateRunInfo() {
+  if (!runToTagInfo)
```

Comment:
optional: prefer to write `{` on such `if` statement (there is a conformance check on this on TypeScript side so we may want to get into habit of doing that)

Reply:
Done.

I much prefer dropping curly for one-line `if`s, though :/